src/sections/Kanvas/kanvas-seo.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+
+const KanvasSEO = () => (
+  <>
+    <Helmet>
+      {/* --- Invisible SEO Improvements --- */}
+      <title>Kanvas | Cloud Native Visual Designer for Kubernetes | Layer5</title>
```

Comment:
a separate page for seo seems redundant? It can be handled in index page only because index page is only crawled by bots.